platforms/hyperledger-fabric/configuration/molecule/crypto/verify.yml

```diff
@@ -317,16 +317,16 @@
 
 #####################################################################################
   #This task checks if genesis block is created or not
-  - name: Check if the genesis block is created or not
-    stat:
-      path: "./build/channel-artifacts/genesis.block.base64"
-    register: genesis_check
+  #- name: Check if the genesis block is created or not
+    #stat:
+      #path: "./build/channel-artifacts/genesis.block.base64"
+    #register: genesis_check
   
 #####################################################################################
   #This task assert if  genesis block is created or not
-  - name: Assert if the genesis block is created or not
-    assert:
-      that: genesis_check.stat.exists
+  #- name: Assert if the genesis block is created or not
```

Comment:
You don't need to comment the code. As this piece of code is irrelevant, you can just remove this piece of code.